spec/pragma.dd

```diff
@@ -144,6 +144,11 @@ $(H3 $(LNAME2 mangle, $(D pragma mangle)))
     $(P There must be one $(ASSIGNEXPRESSION) and it must evaluate at compile time to a string literal.
     )
 
+    $(IMPLEMENTATION_DEFINED On Mac OSX, an extra underscore (`_`) is prepended to the string since 2.079,
```

Comment:
Not sure what the rest of the documentation says but the official name by Apple is `macOS`.